shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/App.kt

```diff
@@ -92,7 +97,28 @@ fun App(context: ApplicationContext) {
                             }
                         }
                         composable<AboutAppScreen> {
-                            AboutApp()
+                            // TODO add action handlers
```

Comment:
please log an issue for TODOs

Reply:
https://github.com/JetBrains/kotlinconf-app/issues/233